### PR TITLE
Playbook for enabling origin-metrics

### PR DESCRIPTION
**Pull request by vnugent**
This PR automates the install steps outlined in origin-metrics [readme](https://github.com/openshift/origin-metrics/blob/master/README.adoc)

### CODE REVIEW

**Comment by detiber** [comment]
@vnugent would you mind refactoring this into an ansible role (similar to openshift_router or openshift_registry) instead?

File: playbooks/adhoc/metrics_setup/playbooks/install.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- name: "Install metrics" 
+  hosts: masters
+  vars:
+     metrics_public_url: "https://{{ metrics_external_service }}/hawkular/metrics"
+  tasks:
+  - name: "Add metrics url to master config"
+    lineinfile: "state=present dest=/etc/origin/master/master-config.yaml regexp='^\ \ metricsPublicURL' insertbefore='^\ \ publicURL' line='\ \ metricsPublicURL: {{ metrics_public_url }}'"
+
+  - name: "Restart master service"
+    service: name=atomic-openshift-master state=restarted
```
Comment (chunyunchen) [inline playbooks/adhoc/metrics_setup/playbooks/install.yml]: For HA master, the master service name is "atomic-openshift-master-api"

**Comment by jeremyeder** [comment]
Comments from @detiber 

- it doesn't integrate with the existing roles for setting the master config
- I don't see a way to configure the SSL certificate for metrics
- it doesn't integrate with the pv/pvc creation we are doing for persistent storage.

**Comment by detiber** [comment]
ok to test

**Comment by detiber** [comment]
@brenton :+1: for adding this as an adhoc playbook until we have a chance to get this converted to a role.

**Pull request merged**